service/controller/clusterapi/v31/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -457,6 +504,12 @@ Resources:
         FromPort: 2379
         ToPort: 2379
         CidrIp: 10.1.0.0/16
+      -
+        Description: Allow traffic from Cluster CIDR to 2379 for network-lb health checks.
```

Comment:
Does `Cluster` mean Tenant Cluster? Could we clarify this? We also talk about Control Plane in the other rules.

Reply:
Good point I will improve comments.